File: Ruby/Queries/ArrayOfMultiples.c
#include <stdio.h>

int main()
{
    int num, length = 0;
    scanf("%d",& num);
    int arr[num];
    for(int i = 1; i <= num; i++){
        if(num % i == 0){
            arr[length] = i;
            printf("%d, ", arr[length]);
            length ++;
        }
    }
    return 0;
}

File: Ruby/Queries/ArrayOfMultiples.rb
puts "Enter the no of which you want to find factors"
num = gets.to_i
length = 0
arr = []

(1..num).each do |i|
  if num % i == 0
    arr[length] = i
    print "#{', ' if length > 0}#{arr[length]}"
    length += 1
  end
end
